physcraper/__init__.py

```diff
@@ -3397,9 +3182,9 @@ def loop_for_write_blast_files(self, key):
         for otu_id in self.sp_d[key]:
             if '^physcraper:status' in otu_id and otu_id['^physcraper:status'].split(' ')[0] not in self.seq_filter:
                 if otu_id['^physcraper:last_blasted'] != '1800/01/01':  # old seq
-                    tax_name = self.ids.find_name(otu_dict_entry=otu_id)
+                    tax_name = self.ids.find_name_otu(otu_dict_entry=otu_id)
                     for tax_name_aln, seq in self.data.aln.items():
-                        otu_dict_name = self.ids.find_name(otu_dict_entry=self.data.otu_dict[tax_name_aln.label])
+                        otu_dict_name = self.ids.find_name_otu(otu_dict_entry=self.data.otu_dict[tax_name_aln.label])
```

Comment:
I'm confused about what this line is doing  - what information are we trying to get here?

Reply:
We are getting the name from an otu_dict_entry. We do not use it though - a dead line of code...

Reply:
Ah ok. So is what we would actually like to happen in that function is to go from the sp_d, which is {taxon_id:[list of otus]}, to the sp_seq_d which is {taxon_id: [list of sequences]} ?

Reply:
It is a helper function for the function write_filterblast_files(), which is writing the local blast files.